tests/base/test_training_data.py

```diff
@@ -332,3 +334,10 @@ def test_training_data_conversion(tmpdir, data_file, gold_standard_file,
     # to dump to the file and diff using git
     # with io.open(gold_standard_file) as f:
     #     f.write(td.as_json(indent=2))
+
+
+def test_url_data_format():
+    test_url = 'http://website-demo.rasa.com/api/default/data.json?api_token=e8c436157a24b007ed5cdc9ba4d062ed66c4f63d'
+    r = requests.get(test_url)
+    data = r.json()
```

Comment:
we don't need to do the request right? can't we just put the value in data?

(the tests should be able to run as independent if possible, in this case this test would randomly start failing if, e.g. the demo is not available, is getting restarted, or if the token changed).

Reply:
Yes, I was wondering about this.  And actually now that I think about it some more, this doesn't actually test much besides whether the link is working.  I'll come up with a better test.